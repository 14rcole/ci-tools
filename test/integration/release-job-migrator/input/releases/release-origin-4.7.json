{
    "name":"4.7.0-0.ci",
    "to": "release",
    "message": "This release contains CI image builds of all code in release-4.7 (master) branches, and is updated each time someone merges.",
    "mirrorPrefix": "4.7",
    "expires":"72h",
    "maxUnreadyReleases": 2,
    "minCreationIntervalSeconds": 1800,
    "pullSecretName": "source",
    "publish":{
      "tag":{"tagRef":{"name":"4.7-ci"}}
    },
    "verify":{
      "upgrade":{
        "upgrade":true,
        "optional":true,
        "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-upgrade-4.7"}
      }
    },
    "periodic":{
      "upgrade-gcp":{
        "interval":"6h",
        "upgrade":true,
        "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-upgrade-4.7"}
      }
    }
  }
