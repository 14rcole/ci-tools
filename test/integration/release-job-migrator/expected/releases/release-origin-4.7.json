{
    "name":"4.7.0-0.ci",
    "to": "release",
    "message": "This release contains CI image builds of all code in release-4.7 (master) branches, and is updated each time someone merges.",
    "mirrorPrefix": "4.7",
    "expires":"72h",
    "maxUnreadyReleases": 2,
    "minCreationIntervalSeconds": 1800,
    "pullSecretName": "source",
    "publish":{
      "tag":{"tagRef":{"name":"4.7-ci"}}
    },
    "verify":{
      "upgrade":{
        "upgrade":true,
        "optional":true,
        "prowJob":{"name":"periodic-ci-openshift-release-master-origin-4.7-e2e-gcp-upgrade"}
      }
    },
    "periodic":{
      "upgrade-gcp":{
        "interval":"6h",
        "upgrade":true,
        "prowJob":{"name":"periodic-ci-openshift-release-master-origin-4.7-e2e-gcp-upgrade"}
      },
      "upgrade-gcp-minor":{
        "interval":"6h",
        "upgrade":true,
        "upgradeType": "PreviousMinor",
        "prowJob":{"name":"release-openshift-origin-installer-e2e-gcp-upgrade-4.6-stable-to-4.7"}
      }
    }
  }
